jina/peapods/pods/__init__.py

```diff
@@ -227,6 +227,8 @@ def _set_conditional_args(args):
                 args.runtime_cls = 'RESTRuntime'
             else:
                 args.runtime_cls = 'GRPCRuntime'
+        if 'parallel' in args and args.parallel == 1:
```

Comment:
parallel is now guaranteed to be in args

Reply:
and it would be good to add a warning

Reply:
> parallel is now guaranteed to be in args
not in case the gateway is added

Reply:
ok, is a smell but good point

Reply:
> and it would be good to add a warning

Thanks, good idea!
However, this is actually expected behaviour:
The documentation for `--separated-workspace` in `mixin_zed_runtime_parser` says: `the data and config files are separated for each pea in this pod, only effective when BasePod\'s parallel > 1`.
In addition there is no logging used in `jina/peapods/pods/__init__.py` for all similar cases.